README.md

```diff
@@ -51,27 +51,27 @@ class app_widgetContainer {
 }
 ```
 
-For methods, follow the standard Apple convention of referring to the first parameter in the method name:
+For methods, when the first argument is part of a prepositional phrase, make the preposition the first label and use the actual argument name inside the method definition:
 
 ```swift
 class Guideline {
-  func combineWithString(incoming: String, options: Dictionary?) { ... }
-  func upvoteBy(amount: Int) { ... }
+  func combine(with incoming: String, options: Dictionary?) { ... }
+  func upvote(by amount: Int) { ... }
```

Comment:
I oftentimes encounter situations where I want additional context on a parameter name. For example:

`func item(with name: String) -> Item?` should probably be `func item(withName name: String) -> Item?`

This is helpful when the purpose of the parameter isn't directly inferrable by the preposition label or type type of the parameter (e.g., "Is this string a name? An ID?").